doot-doot/cogs/airhorn.py

```diff
@@ -1,33 +1,38 @@
 import discord
+import random
 import asyncio
 from discord.ext import commands
 
 async def play_file(ctx, filename):
-     voice_channel = ctx.author.voice.channel
-     print(f'{str(ctx.author)} is in {voice_channel}')
-     try:
-         voice_channel = await voice_channel.connect()
-     except:
-         voice_channel.disconnect()
-         ctx.send("Exception occured, automatic process atempted to repair it, please try again.")
+    voice_channel = ctx.author.voice.channel
+    print(f'{str(ctx.author)} is in {voice_channel}')
+    try:
+        voice_channel = await voice_channel.connect()
+    except Exception as e:
+        voice_channel.disconnect()
+        ctx.send("Exception occured, automatic process " +
+        "atempted to repair it, please try again. | " + str(e))
 
-     source = discord.FFmpegPCMAudio(filename)
+    # There is a 1 in 100th chance that it
+    # will do a rickroll instead of the desired sound
+    random = random.randint(1, 100)
+    if random == 1:
```

Comment:
Variable name random caused my IDE to scream about variable being used before being assigned (because of line ```random = random.randint(1, 100)``` while i didnt test if it work anyway for QoL i'd love if you could change var name to i.e random_chance
ks